test/browser/rowHeight.test.ts

```diff
@@ -51,3 +60,47 @@ test('rowHeight is function', async () => {
   await userEvent.keyboard('{Control>}{end}');
   expect(grid.scrollTop + grid.clientHeight).toBe(grid.scrollHeight);
 });
+
+test('rowHeight with repeat pattern - multiple identical heights', () => {
+  expectGridRows(() => 40, 'repeat(1, 35px) repeat(50, 40px)');
```

Comment:
Why do we always have `repeat(1, 35px)`?
I understand it's for the header row, but can't it just be `35px`?